kyc_account/models/res_partner.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    def _kyc_accept_transaction(self, _record, raise_if_not=True):
+        self.ensure_one()
+        if (
+            _record
+            and _record[0]._name == "account.move"
```

Comment:
you have a `ensure_one`, no need to do `_record[0]` as you already ensured that you have only one.

Reply:
_record are journals in this case and that could be multiple